torch/_tensor_docs.py

```diff
@@ -4845,6 +4845,49 @@ def callable(a, b) -> number
 ``x.T`` is equivalent to ``x.permute(n-1, n-2, ..., 0)``.
 """)
 
+add_docstr_all('H',
+               r"""
+Returns a view of a matrix (2-D tensor) conjugated and transposed.
+
+``x.H`` is equivalent to ``x.transpose(0, 1).conj()`` for complex matrices and
+``x.transpose(0, 1)`` for real matrices.
+
+.. seealso::
+
+        :attr:`~.Tensor.mH`: An attribute that also works on batches of matrices.
+""")
+
+add_docstr_all('mT',
+               r"""
+Is this tensor with the last two dimensions transposed.
```

Comment:
"Returns a view of its tensor with the last two dimensions transposed"?

Reply:
I just went with the previous wording that `.T` had, but I admit it's quite weird. I'll update it to yours.